hack/e2e-test-ocp.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# Copyright 2025 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export CWD=$(pwd)
+
+function cert_manager_deploy {
+      oc apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.17.0/cert-manager.yaml
+      oc -n cert-manager wait --for condition=ready pod -l app.kubernetes.io/instance=cert-manager --timeout=2m
+}
+
+function lws_deploy {
+    if [ -z "$RELEASE_IMAGE_LATEST" ]; then
+      echo "RELEASE_IMAGE_LATEST is empty"
+      exit 1
+    fi
+    if [ -z "$KUBECONFIG" ]; then
+      echo "KUBECONFIG is empty"
+      exit 1
+    fi
+    if [ -z "$NAMESPACE" ]; then
+      echo "NAMESPACE is empty"
+      exit 1
+    fi
+
+echo "apiVersion: config.lws.x-k8s.io/v1alpha1
+kind: Configuration
+internalCertManagement:
+  enable: false
+leaderElection:
+  leaderElect: true" > "$CWD"/config/manager/controller_manager_config.yaml
+
+    pushd "$CWD"/config/manager
+      echo "Setting the kubernetes-sigs-lws image built from this version"
+      # take the domain name of the cluster
+      REGISTRY=$(echo "$RELEASE_IMAGE_LATEST" | awk -F'/' '{print $1}')
+      IMAGE_TAG=$REGISTRY/$NAMESPACE/pipeline:kubernetes-sigs-lws
```

Comment:
we should be able to pass this directly from o/release

Reply:
As far as I know, this is valid for components that are in release payload. For the optional operators, it might not be possible. I mimicked from https://github.com/openshift/secondary-scheduler-operator/blob/07a1dd81bc8f56f4b34427b0c8f055d6ba3485b4/test/e2e/operator_test.go#L118-L120 but I'd be very happy if there is more elegant way because I'm copying this string parsing everywhere :)